packages/webcert/src/feature/certificate/Inputs/UeTextField.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import * as redux from 'react-redux'
+import userEvent from '@testing-library/user-event'
+import { fakeTextFieldElement } from '@frontend/common'
+import UeTextField from './UeTextField'
+
+const useSelectorSpy = jest.spyOn(redux, 'useSelector')
+const useDispatchSpy = jest.spyOn(redux, 'useDispatch')
+
+useSelectorSpy.mockReturnValue({})
+useDispatchSpy.mockReturnValue(jest.fn())
+
+const mockQuestion = {
+  type: fakeTextFieldElement({ id: '1' })['1'],
+}
```

Comment:
Please replace with `const mockQuestion = fakeTextFieldElement({ id: '1' })['1']`. `type` is not necessary here.